fire/api/model/punkttyper.py

```diff
@@ -281,6 +281,21 @@ def landsnummer(self) -> str:
 
         return self.ident
 
+    @property
+    def gnssnavn(self) -> str:
+        gnssnavne = []
```

Comment:
Jeg er ikke sikker på, at vi gør det konsistent, men det virker rarere for mine øjne at læse `gnss_navn` frem for `gnssnavn`. Mangler at kunne markere forslag som 'optional' .